packages/menu/src/Menu.ts

```diff
@@ -575,55 +558,34 @@ export class Menu extends SizedMixin(SpectrumElement, { noDefaultSize: true }) {
         }
     }
 
-    protected navigateWithinMenu(event: KeyboardEvent): void {
-        const { key } = event;
-        const lastFocusedItem = this.childItems[this.focusedItemIndex];
-        const direction = key === 'ArrowDown' ? 1 : -1;
-        const itemToFocus = this.focusMenuItemByOffset(direction);
-        if (itemToFocus === lastFocusedItem) {
-            return;
-        }
-        event.preventDefault();
-        event.stopPropagation();
-        itemToFocus.scrollIntoView({ block: 'nearest' });
-    }
-
-    protected navigateBetweenRelatedMenus(event: KeyboardEvent): void {
-        const { key } = event;
+    protected navigateBetweenRelatedMenus(event: MenuItemKeydownEvent): void {
```

Comment:
Since menu items have focus, menuitems need to forward keyboard event info to manage this.